City.h

```diff
@@ -0,0 +1,57 @@
+#include "Area.h"
+class City{
+    private:
+    vector<Area> areas;
+    string nameCity;
+    public:
+    void printCity(){
+        cout << "City:" << nameCity << endl;
+    }
+
+    City(string tmpName){
+        nameCity = tmpName;
+    }
+
+    void addArea(Area  area){
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.